client/js/lounge.js

```diff
@@ -417,7 +417,7 @@ $(function() {
 		nicks = [];
 
 		for (i in data.users) {
-			nicks.push(data.users[i].name);
+			nicks.push(data.users[i].nick);
```

Comment:
So, was name just unnecessary? We were mapping it to nick and using it for ... what reason?